files/angular-pouchdb/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-pouchdb",
+  "repo": "angular-pouchdb/bower-angular-pouchdb",
+  "files": {
+    "include": ["angular-pouchdb.js", "angular-pouchdb.min.js"]
```

Comment:
Also `angular-pouchdb.min.js` doesnt exist here https://github.com/angular-pouchdb/angular-pouchdb/tree/v3.0.0
The bot will fail